README.md

```diff
@@ -12,17 +13,23 @@
 - Run "./npm i @angular/cli"
 - Run "./npm i"
 - Run "./mvnw generate-resources"
+
+# IntelliJ
 - Open the project in intellij
-- Build src/main/java/AuthenticationProxyApplication.java
-- Run src/main/java/AuthenticationProxyApplication.java
+- Change the user to postgres and password to the password for that superuser in application.properties
+- Build and run src/main/java/AuthenticationProxyApplication.java
 
-- ! For the application to run the capstone_user must have permission to SELECT from both tables !
+# pgAdmin
+- Go into pgAdmin, navigate to the capstone_user user under Login/Group Roles under the PostgreSQL 15 server. 
+- Right-click and select Properties, go to the Privileges tab, and enable the 'Can initiate streaming replication and backups?' privilege, then click Save
```

Comment:
You no longer need "can initiate streaming replication and backups" I made sure it worked without it